lstchain/reco/volume_reducer.py

```diff
@@ -75,7 +75,6 @@ def check_and_apply_volume_reduction(event, config):
         pass
 
     else:
-        print("Volume reduction algorithm being used: {}".format(volume_reduction_algorithm))
```

Comment:
Not really against or in favor (just matter of tastes). But why taking out the stdout message ? It is true that to apply the vol_reduc, you must had consciously modified the config file... But this would be just a double check of what are you doing/applying int the analysis

Reply:
There shouldn't be and prints in the code base. If you want to keep this message, use the logging module (debug or info for this)